storage_service/locations/views.py

```diff
@@ -44,8 +44,23 @@ def get_delete_context_dict(request, model, object_uuid, default_cancel='/'):
 
 def package_list(request):
     packages = Package.objects.all()
+
+    # Amalgate last fixity check date and result with package data
+    package_data = []
+    for package in packages:
+        fixity_checks = FixityLog.objects.filter(package=package).order_by('-datetime_reported')
+        latest_check = fixity_checks[0].datetime_reported if fixity_checks else None
+        latest_result = fixity_checks[0].success if fixity_checks else None
+        package_data.append((latest_check, latest_result, package))
```

Comment:
I see you added the fixity attributes to the `Package` class, this view should be updated to use them instead of re-generating the information.  The template also needs to be updated for the new attributes.  `render` should return the `context` that it uses, not `locals()`.